client.go

```diff
@@ -348,6 +348,11 @@ func (c *Client) Transport() TransportInfo {
 	return c.transport
 }
 
+// Info returns client metadata set by a handler or encoded into the token.
+func (c *Client) Info() Raw {
+	return append(make(Raw, 0, len(c.info)), c.info...)
```

Comment:
c.info can be changed from inside refresh handler/callback, so this should be protected by mutex.